packages/marketplace/contracts/OrderValidator.sol

```diff
@@ -22,20 +22,18 @@ contract OrderValidator is IOrderValidator, Initializable, EIP712Upgradeable, Wh
 
     /// @notice initializer for OrderValidator
     /// @param admin OrderValidator and Whiteist admin
-    /// @param newTsbOnly boolean to indicate that only The Sandbox tokens are accepted by the exchange contract
-    /// @param newPartners boolena to indicate that partner tokens are accepted by the exchange contract
-    /// @param newOpen boolean to indicate that all assets are accepted by the exchange contract
-    /// @param newErc20 boolean to activate the white list of ERC20 tokens
+    /// @param roles of the Whitelist contract
+    /// @param permissions of the roles
+    /// @param whitelistsEnabled boolean to indicate id whitelist is enabled 
```

Comment:
fix natspec spacing

+ i don't understand the sentence
"id whitelist is enabled " ?

Reply:
here we check 'isWhitelistsEnabled'